projects/dsp-ui/src/lib/action/directives/external-links/external-links.directive.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { ExternalLinksDirective } from './external-links.directive';
+
+@Component({
+    template: `<a href="http://dasch.swiss/">{{ label }}</a>`
+})
+class TestExternalLinkComponent {
+
+    label = 'DaSCH website';
+
+    constructor() { }
+}
+
+describe('ExternalLinksDirective', () => {
+
+    let testHostComponent: TestExternalLinkComponent;
+    let testHostFixture: ComponentFixture<TestExternalLinkComponent>;
+    let linkEl: DebugElement;
+
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [
+                ExternalLinksDirective,
+                TestExternalLinkComponent
+            ]
+        });
+
+        testHostFixture = TestBed.createComponent(TestExternalLinkComponent);
+        testHostComponent = testHostFixture.componentInstance;
+        testHostFixture.detectChanges();
+
+        linkEl = testHostFixture.debugElement.query(By.css('a[href]'));
+
+        expect(testHostComponent).toBeTruthy();
+
+    });
+
+    it('should create an instance', () => {
+        expect(testHostComponent).toBeTruthy();
+    });
+
+    it('should get the correct attributes', () => {
```

Comment:
Maybe you could add another test for an internal links and check the "_blank" is not added.